cartography/graph/job.py

```diff
@@ -86,6 +124,36 @@ def from_json(cls, blob: str, short_name: Optional[str] = None) -> 'GraphJob':
         name = data["name"]
         return cls(name, statements, short_name)
 
+    @classmethod
+    def from_node_schema(cls, node_schema: CartographyNodeSchema, parameters: Dict[str, Any]) -> 'GraphJob':
```

Comment:
rename to `cleanup_job_from_node_schema`

Reply:
There is also a `GraphJob.from_json_file()`, so I think we should follow the naming convention